x-pack/plugins/data_enhanced/server/search/session/check_running_sessions.ts

```diff
@@ -154,7 +154,7 @@ export async function checkRunningSessions(
   try {
     await getAllSavedSearchSessions$(deps, config)
       .pipe(
-        mergeMap(async (runningSearchSessionsResponse) => {
+        concatMap(async (runningSearchSessionsResponse) => {
```

Comment:
Would be nice to adjust unit tests to take this new seq. behavior into account

Reply:
I am adding it here - https://github.com/elastic/kibana/pull/96321
IMO feel free to skip